tool/tctl/common/plugins_command.go

```diff
@@ -45,10 +59,27 @@ func (p *PluginsCommand) Initialize(app *kingpin.Application, config *servicecfg
 	p.dryRun = true
 
 	pluginsCommand := app.Command("plugins", "Manage Teleport plugins.").Hidden()
+
 	p.cleanupCmd = pluginsCommand.Command("cleanup", "Cleans up the given plugin type.")
 	p.cleanupCmd.Arg("type", "The type of plugin to cleanup. Only supports okta at present.").Required().EnumVar(&p.pluginType, string(types.PluginTypeOkta))
 	p.cleanupCmd.Flag("dry-run", "Dry run the cleanup command. Dry run defaults to on.").Default("true").BoolVar(&p.dryRun)
 
+	p.install.cmd = pluginsCommand.Command("install", "Install new plugin instance")
+
+	p.install.scim.cmd = p.install.cmd.Command("scim", "Install a new SCIM integration")
```

Comment:
How do you remove this plugin? Is there a corresponding "delete" CLI command?

Also, when it's created, how does the integrations page in the web UI behave?

Reply:
<img width="865" alt="Screenshot 2024-05-13 at 9 03 52 PM" src="https://github.com/gravitational/teleport/assets/1959769/b2d1c6f1-9c68-414d-9654-9cc56ee9ac36">